api/API.md

```diff
@@ -196,6 +196,105 @@ The `config` endpoint, specifically the `EdgeDevConfig` message, supports arbitr
 
 The `EdgeDevConfig` message can contain zero, one or more `ConfigItem` entries, each of which is an arbitrary key/value pair. These SHOULD be used to send implementation-specific configuration to a Device, other than configuration already defined in this API or the messages. The items are defined in [configuration properties](../docs/CONFIG-PROPERTIES.md)
 
+### Controller Certificates
+
+Retrieve Certificates that Controller would use for signing or any key exchanges. Controller can include one or more certificates in the response, and include information about each certificate such as unique identifier for the certificate, the target usage for the certificate and any other properties related to the certificate. Each certificate MUST be a X.509 certificate in PEM format.
```

Comment:
Please add blank line after first sentence, since that sentence is basically a sub-title to the section.

"would use" is "will use" or "are using".

"or any  key exchanges" doesn't quite parse.

Reply:
Updated.